server/jms/src/main/java/org/hawkular/apm/server/jms/CommunicationDetailsCacheMDB.java

```diff
@@ -30,17 +30,20 @@
 import com.fasterxml.jackson.core.type.TypeReference;
 
 /**
+ * This MDB is provided to populate the CommunicationDetailsCache when Communication Details.
+ * Each clustered APM server node will receive the communication details data, so each cache
+ * is expected to only be a local cache.
+ *
+ * Discussion regarding potential future use of a distributed cache is associated with HWKAPM-479.
+ * If this change happens, then this MDB should be converted back to a durable (load balanced) subscriber.
+ *
  * @author gbrown
  */
-@MessageDriven(name = "CommunicationDetails_Cache", messageListenerInterface = MessageListener.class,
+@MessageDriven(name = "CommunicationDetails_CommunicationDetailsCache", messageListenerInterface = MessageListener.class,
         activationConfig =
         {
                 @ActivationConfigProperty(propertyName = "destinationType", propertyValue = "javax.jms.Topic"),
-                @ActivationConfigProperty(propertyName = "destination", propertyValue = "CommunicationDetails"),
-                @ActivationConfigProperty(propertyName = "subscriptionDurability", propertyValue = "Durable"),
```

Comment:
Have read further in the JMS/wildfly docs - if clientId and subscriptionName are the same on multiple nodes, then they will load balance the messages from the durable subscription (topic). However to get all nodes receiving from the topic, but still be durable, I just need to change the clientId - which can apparently be done in the MDB using "Annotation Property Replacement" :https://docs.jboss.org/author/display/WFLY10/Java+EE+Application+Deployment+Configuration

Will need to experiment to see what the issues might be if (for example) a node name is not specified.